joomla/joomla-cms
# [4.0] Center status/date created headers

### Summary of Changes
Center status header to match with its td cell.


### Testing Instructions
Code review.
or
Go to Content > Articles
See Date Created header right-aligned vs. center-aligned in Featured Articles.
Go to Content > Featured Articles
See Status header left-aligned.

(Optional to better see the alignment issue in Smart Search) Install https://github.com/C-Lodder/joomla4-backend-template/releases
Go to System > Administrator Template Styles
Set Bettum the default.

Go to Components > Smart Search > Index
See Status header left-aligned.
Go to Components > Smart Search > Filters
See Status header left-aligned.


### Expected result
Table header/cell respectively should be center-aligned.
